docs/tutorial.rst

```diff
@@ -47,6 +47,11 @@ Create this folder::
 
 The ``.gitignore`` will remain and keep Git from not tracking the directory.
```

Comment:
Just a drive by comment. What does this mean ? :)

Maybe we can improve this paragraph as part of this PR.

Reply:
Nothing to do with me, but this line seems a bit redundant looking at the snippet above